SPECS/maven-surefire/maven-surefire.signatures.json

```diff
@@ -0,0 +1,7 @@
+{
+    "Signatures": {
+     "maven-surefire-3.0.0.tar.gz": "3a8ab67869ac4b83cde22ca23b68140b2b46feb7d970db95e42923eb95f9f908",
+     "cpl-v10.html":"dde4c2b665c81a60a6dec39beb54b89836ae071e552d935a04880f9d5a2e58cd",
+     "generate-tarball.sh":"b223db29e37a7bfd28678bc01b63131e28158e97568ca8d3a4d870256d0bddcf"
```

Comment:
do you need to embed generate-tarball.sh in package ? if not I don't think we need its signature

Reply:
removed. thanks for pointing it out. Fedora folks had this way.